_data-prepper/trace-analytics.md

```diff
@@ -8,40 +8,44 @@ nav_order: 21
+- Add an **instrumentation** to your application so it can generate telemetry data and send it to an OpenTelemetry collector.
+- Run an **OpenTelemetry collector** as a sidecar or daemonset for EKS, a sidecar for ECS, or an agent on EC2. You should configure the collector to export trace data to Data Prepper. 
+- Deploy **Data Prepper** as an intermediate component and configure it to send the enriched trace data to your OpenSearch cluster or Amazon OpenSearch Service domain. 
+- Use **OpenSearch Dashboards** to visualize and detect problems in your distributed applications.
 
 ## Trace analytics pipeline
 
 To achieve trace analytics in Data Prepper, we have three pipelines: `entry-pipeline`, `raw-trace-pipeline` and `service-map-pipeline`.
 
-![Trace analytics pipeline]({{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-feature.jpeg)
+<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-feature.jpg" alt="Trace analyticis pipeline overview">{: .img-fluid}
 
 
 ### OpenTelemetry trace source
 
-The [OpenTelemetry source](../data-prepper-plugins/otel-trace-source/README.md) accepts trace data from the OpenTelemetry collector. The source depends on [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/protocol). The source officially support transport over gRPC. The source also supports industry-standard encryption (TLS/HTTPS).
+The [OpenTelemetry source]({{site.url}}{{site.baseurl}}/data-prepper-plugins/otel-trace-source/README.md) accepts trace data from the OpenTelemetry collector. The source follows the [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/protocol). The source officially supports transport over gRPC and the use of industry-standard encryption (TLS/HTTPS).
 
 ### Processor
 
 There are two processors for the Trace Analytics feature:
 
-* *otel_trace_raw* -  This is a processor that receives collection of [Span](../../data-prepper-api/src/main/java/com/amazon/dataprepper/model/trace/Span.java) records sent from [otel-trace-source](../dataPrepper-plugins/otel-trace-source), and performs stateful processing on extracting and filling-in trace group related fields.
-* *otel_trace_group* -  This is a processor that fills in the missing trace group related fields in the collection of [Span](../../data-prepper-api/src/main/java/com/amazon/dataprepper/model/trace/Span.java) records by looking up the OpenSearch backend.
+* *otel_trace_raw* -  This is a processor that receives collection of [Span]({{site.url}}{{site.baseurl}}/data-prepper-api/src/main/java/com/amazon/dataprepper/model/trace/Span.java) records sent from [otel-trace-source]({{site.url}}{{site.baseurl}}/dataPrepper-plugins/otel-trace-source), and performs stateful processing on extracting and filling-in trace group related fields.
```

Comment:
> ({{site.url}}{{site.baseurl}}/data-prepper-api/src/main/java/com/amazon/dataprepper/model/trace/Span.java

This needs to be the URL to GitHub since we are linking the user to the actual Java class model.

> {{site.url}}{{site.baseurl}}/dataPrepper-plugins/otel-trace-source

This should also be `/docs/2.4/data-prepper/configuration/sources/otel-trace/`. I'm not sure how the `docs/2.4` operates here, so you might need to modify slightly.